utilities/utilities.py

```diff
@@ -348,19 +353,21 @@ def check_for_new_youtube_comments(self):
         return new_comments
 
     def get_question(
-        self, order_type="TOP", wiki_question_bias=SemanticWiki.default_wiki_question_percent_bias
+        self, order_type: OrderType = OrderType.TOP, wiki_question_bias: float =SemanticWiki.default_wiki_question_percent_bias
```

Comment:
anything wrong about type inference?

Reply:
IMO it's a good practice to type hint arguments so that there's no ambiguity wrt their types. Maybe I'm excessively pedantic in that regard but from my experience this prevents a lot of runtime errors and increases code readability.